.rubocop.yml

```diff
@@ -1,7 +1,10 @@
 require: umts-custom-cops
 inherit_from: .rubocop-no-umts-cops.yml
+
 # Hold on there cowpoke! Don't write them rules here!
 # Write them in .rubocop-no-umts-cops.yml!
 # Otherwise any setting here will be respected by your local Rubocop,
 # but not by CodeClimate.
 # This is because CodeClimate is not capable of running our custom cops.
+UmtsCustomCops/BeMatcherForNonDuplicableTypes:
+  Enabled: false
```

Comment:
No, it specifically shouldn't.

This cop is only availabe to disable if we are using umts-custom-cops. If I were to add this rule to `.rubocop-no-umts-cops.yml`, then rubocop on Travis would produce an error because the cop doesn't exist.

Reply:
BTW, see umts/custom-cops#9